types/react-phone-number-input/index.d.ts

```diff
@@ -34,6 +34,12 @@ export function isPossiblePhoneNumber(value: string): boolean;
  */
 export function isValidPhoneNumber(value?: string): boolean;
 
+/**
+ * Checks if a country is supported by this library.
+ * This is simply an alias for isSupportedCountry() from libphonenumber-js.
+ */
+export function isSupportedCountry(iso: string): boolean;
```

Comment:
Maybe we should use the same signature as in library docs:
https://github.com/catamphetamine/react-phone-number-input/blob/master/README.md#issupportedcountrycountry-string-boolean

```suggestion
export function isSupportedCountry(country: string): boolean;
```